ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1202,3 +1202,14 @@ literals:
 - thermal_energy_accumulator
 - ground_isolation_command
 - ground_temperature_sensor
+
+# New Fields for Drain Pumps
+- condensate_water_discharge_pump_run_command
```

Comment:
what are these part of? if this is for modeling a pump, almost everything except for run_command is unnecessary. same for the other fields defined here.

Reply:
condensate_water_discharge_pump_run_command,  condensate_water_discharge_pump_run_status  and condensate_water_discharge_pump_failed_alarm  are part of the drain pumps. 
We have considered asset as PMP (Pump) and will be modeling the drain pump as "Pump".

Reply:
then these fields and most of this CL can be closed. please make those changes.

Reply:
The points have been updated in the Abstract and new canonical type